.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release to GitHub Container Registry
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Build Docker image
+      run: docker build -t model-serve .
+
+    - name: Log into GitHub Container Registry
+      run: echo "${{ secrets.TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+    - name: Tag Docker image
+      run: docker tag model-serve ghcr.io/remla2024-team14/model-serving:latest
+
+    - name: Push Docker image to GitHub Container Registry
+      run: docker push ghcr.io/remla2024-team14/model-serving:latest
```

Comment:
Here you could use `docker push --all-tags $IMG` which will push all tags of that image in one go to the repo

Reply:
You may also consider semantic versioning as was shown in the tutorial "delivery-with-github-in-class"

<img width="1089" alt="image" src="https://github.com/remla2024-team14/model-service/assets/74115586/887d7fbd-a46e-4a72-a1a6-c86e53ca0dfe">